server/db.js

```diff
@@ -127,17 +133,6 @@ exports.getUserById = function(userId, callback){
     getUserById(userId, callback);
 }
 
-var getUserById = function(userId, callback) {
```

Comment:
Why Is this function deleted? You are still calling getUserById(userId, callback); on line 127

Reply:
Cuz it's a duplicate. It turns out there another function that does the same thing with the same name. Dups are bad :(

Reply:
true